torchvision/features/_core.py

```diff
@@ -0,0 +1,13 @@
+from typing import Any
+
+import torch
+
+__all__ = ["Feature"]
+
+
+class Feature(torch.Tensor):
```

Comment:
This is great!

One meta-question that the team needs to figure out is if we will be supporting torchscript for those features or not. IIUC this might not be propagating the inherited type to torchscripted functions

Reply:
I've split this into `Feature` and `TensorFeature`, since a `Feature` might not necessarily be a `Tensor`. For example, `Text` cannot be represented, but could very well be regarded as `Feature` from `torchtext`.